examples/varteleport.qasm

```diff
@@ -27,7 +27,7 @@ rz(pi / 4) input;
 
 let io = input;
 for i in [0: n_pairs - 1] {
-  let bp = q[2 * i, 2 * i + 1];
+  let bp = q[2 * i: 2 * i + 1];
```

Comment:
The comma syntax is valid--see https://qiskit.github.io/openqasm/language/types.html#register-concatenation-and-slicing. it picks out the particular entries of interest. I think they are actually equivalent in this case, as the range is inclusive.